lnrpc/routerrpc/router_backend.go

```diff
@@ -1191,6 +1205,22 @@ func (r *RouterBackend) extractIntentFromSendRequest(
 		payIntent.DestFeatures = features
 	}
 
+	// Validate that the MPP parameters are compatible with the
+	// payment amount. In other words, the parameters are invalid if
```

Comment:
why aren't we doing this check also where we check the MaxShardParts ? We already have all the infos available earlier ?

Reply:
gottya